src/Rsk.DuendeIdentityServer.AuditEventSink/Adapters/TokenIntrospectionSuccessEventAdapter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Duende.IdentityServer.Events;
+using RSK.Audit;
+
+namespace Rsk.DuendeIdentityServer.AuditEventSink.Adapters
+{
+    public class TokenIntrospectionSuccessEventAdapter : IAuditEventArguments
+    {
+        private readonly TokenIntrospectionSuccessEvent evt;
+
+        public TokenIntrospectionSuccessEventAdapter(TokenIntrospectionSuccessEvent evt)
+        {
+            this.evt = evt ?? throw new ArgumentNullException(nameof(evt));
+        }
+
+        public ResourceActor Actor => new ResourceActor(ResourceActor.UserSubjectType, evt.ActivityId, evt.ApiName);
```

Comment:
Here, I don't think we have a Resource Actor identifier. Should this be null or empty?